src/SizeBench.LocalBuild/LocalBuildPathLocator.cs

```diff
@@ -40,4 +40,30 @@ public bool TryInferBinaryPathFromPDBPath(string pdbPath, out string binaryPath)
         binaryPath = String.Empty;
         return false;
     }
-}
\ No newline at end of file
+
+    public bool TryInferPDBPathFromBinaryPath(string binaryPath, out string pdbPath)
+    {
+        if (binaryPath is null)
+        {
+            pdbPath = String.Empty;
+            return false;
```

Comment:
Nitpick: No need to check for null, this code now supports C# nullable reference types, so the `string binaryPath` not being `string?` implies we don't support nulls anyway.

Reply:
I'm snowcloning the pattern of `TryInferBinaryPathFromPDBPath` that checks for null.